src/Microsoft.AspNet.Mvc.ModelBinding/Validation/DataAnnotationsClientModelValidatorProvider.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding.Validation
+{
+    /// <summary>
+    /// An implementation of <see cref="IClientModelValidatorProvider"/> which provides client validators
+    /// for attributes which derive from <see cref="ValidationAttribute"/>. It also provides
+    /// a validator for types which implement <see cref="IClientModelValidator"/>.
+    /// The logic to support <see cref="IClientModelValidator"/>
+    /// is implemented in <see cref="DataAnnotationsClientModelValidator{}"/>.
```

Comment:
bit strange since `DataAnnotationsClientModelValidator` class doesn't exist and empty curly braces should also cause an error.  try removing `{}` then.